config/qualifications.yml

```diff
@@ -1,21 +1,31 @@
-# A fully-trained programmer should
 ---
-title: 'Have demonstrated understanding of Ruby core methods / Rails object extensions'
+title: 'Configure your laptop for development'
```

Comment:
Shouldn't matter one way or the other, but I think you can skip the quotes.

Reply:
That'd be nice anyway